genesyscloud/outbound/resource_genesyscloud_outbound_init.go

```diff
@@ -25,7 +25,7 @@ func SetRegistrar(l registrar.Registrar) {
 	l.RegisterResource("genesyscloud_outbound_messagingcampaign", ResourceOutboundMessagingCampaign())
 	l.RegisterResource("genesyscloud_outbound_sequence", ResourceOutboundSequence())
 	l.RegisterResource("genesyscloud_outbound_settings", ResourceOutboundSettings())
-	l.RegisterResource("genesyscloud_outbound_wrapupcodemappings", resourceOutboundWrapUpCodeMappings())
+	l.RegisterResource("genesyscloud_outbound_wrapupcodemappings", ResourceOutboundWrapUpCodeMappings())
```

Comment:
This was causing one of the issues where the genesyscloud_outbound_wrapupcodemappings resource was not registered so when the export ran, the exporter complained about not being able to find the resource.